Formula/asimov.rb

```diff
@@ -0,0 +1,53 @@
+class Asimov < Formula
+  desc "Automatically exclude development dependencies from Time Machine backups"
+  homepage "https://github.com/stevegrunwell/asimov"
+  url "https://github.com/stevegrunwell/asimov/archive/v0.2.0.tar.gz"
+  sha256 "2efb456975af066a17f928787062522de06c14eb322b2d133a8bc3a764cc5376"
+  revision 0
+
+  # Setup HEAD support (install with --HEAD)
+  head "https://github.com/stevegrunwell/asimov.git", :branch => "develop"
```

Comment:
The comments aren't needed, please group all this metadata on the top of the file